tests/bigrepo.js

```diff
@@ -0,0 +1,34 @@
+const codeClient = require('../dist/index');
```

Comment:
required: I believe this should not be committed?

On the other hand - shall we have an actual performance tests here? I think we learned a lesson that we don't want to accidentally end up in the same situation again.

Reply:
Want to stress the performance tests. I would imagine it will take some extra effort but could be beneficial for us to be more confident in our changes in the future.

Reply:
I thought I've removed this file from the commit. I think we should have a performance test instead (but I'm not sure how to test it properly yet)